material-github.user.css

```diff
@@ -1080,8 +1080,282 @@
+	}
+	button.width-full.text-left.btn-link.f6.muted-link {
+		border-top: none!important;
+	}
+	button.width-full.text-left.btn-link.f6.muted-link:hover {
+		color: var(--selected) !important;
+	}
+
+	/* Home, Recent activity */
+	.js-show-more-recent-items,
+	li.Box-row.d-flex.p-2,
+	li.d-flex.border-top.p-2.position-relative {
+		border-top: 1px solid var(--bg3)!important;
+	}
+	li.Box-row.d-flex.p-2:first-of-type {
+		border-top-color: transparent!important;
+	}
+	.js-details-target.js-show-more-recent-items {
+		background-color: var(--bg2)!important;
+		color: rgba(255, 255, 255, 0.6) !important;
+	}
+	.Box-row--hover-blue:hover {
+		background-color: var(--bg3)!important;
+	}
+	.link-gray:hover {
+		color: var(--selected)!important;
+	}
+
+	/* Home, All activity */
+	.dashboard .js-all-activity-header + div {
+		background-color: var(--bg2) !important;
+		border-color: var(--bg2) !important;
+	}
+	.dashboard-rollup-items .body {
+		border-top: 1px solid var(--bg4) !important;
+	}
+	.js-news-feed-event-group.border-bottom {
+		border-bottom: 1px solid var(--bg4)!important;
+	}
+	.js-news-feed-event-group.border-bottom .d-flex.d-flex.flex-items-baseline.py-3 {
+		border-bottom: none!important;
+	}
+	.d-flex.flex-items-baseline.py-3 {
+		border-bottom: 1px solid var(--bg4)!important;
+	}
+
+	/* Home, Explore repositories */
+	.py-2.my-2.border-bottom {
+		border-color: var(--bg2) !important;
+	}
+
+	/* Profile, Repositories, Type & Language */
+	label.SelectMenu-item[aria-checked=true] {
+		color: rgba(255, 255, 255, 0.7)!important;
+	}
+	label.SelectMenu-item {
+		color: rgba(255, 255, 255, 0.3);
+		background-color: var(--bg3);
+		border-bottom: 1px solid var(--bg4);
+	}
+
+	/* Profile, Overview, Customize your pins */
+	.pinned-item-name {
+		color: rgba(255, 255, 255, 0.3)!important;
+	}
+	.pinned-item-checkbox:checked + .pinned-item-name {
+		background-color: var(--bg3)!important;
+	}
+
+	/* Marketplace, Apps */
+	.topic-tag-outline {
+		box-shadow: inset 0 0 0 1px var(--btn)!important;
+	}
+	.ScreenshotCarousel-nav {
+		box-shadow: inset 0 1px 0 var(--bg3)!important;
+	}
+	.ScreenshotCarousel-navitem {
+		border: 1px solid var(--bg3)!important;
+	}
+	.Label--outline {
+		color: rgba(255, 255, 255, 0.3)!important;
+		border-color: var(--bg6);
+	}
+
+	/* Repo, Header, recent pushes */
+	.Box-row--yellow {
+		background-color: var(--bg3)!important;
+		color: var(--bg5)!important;
+	}
+	.flex-auto {
+		color: rgba(255, 255, 255, 0.7) !important
+	}
+
+	/* Repo, Markdown, checkbox */
+	/* (Pick From "Github Ice Dark(Updated)", by rouzbeh84 + JZersche, from userstyles.org) */
+	input[type="checkbox"],
+	input[type="checkbox"]:focus,
+	input[type="checkbox"]:hover:active,
+	input[type="checkbox"]:disabled,
+	input[type="checkbox"]:hover:disabled,
+	input[type="checkbox"]:hover:active:disabled {
+		-webkit-appearance: none !important;
+		border: 1px solid var(--bg5) !important;
+		height: 1em !important;
+		width: 1em !important;
+	}

```

Comment:
Did you just remove background-color and color properties ? :D

Reply:
I see that would have no effect superficially, just ill-considered maybe.